wolfcrypt/src/aes.c

```diff
@@ -2477,6 +2505,23 @@ static void wc_AesDecrypt(Aes* aes, const byte* inBlock, byte* outBlock)
             return BAD_FUNC_ARG;
         }
 
+    #if defined(WOLFSSL_SCE) && defined(WOLFSSL_RENESAS_RA6M3G)
```

Comment:
I removed "#if defined(WOLFSSL_SCE) && defined(WOLFSSL_RENESAS_RA6M3G)". The other checks weren't there previously.  So when NO_WOLFSSL_AES_192 was defined it would eventually fail on a test because my function would return an BAD_FUNC_ARG error because of the keylen.